src/plugins/blocksmith/components/blocksmith/constants.js

```diff
@@ -1,11 +1,14 @@
 export default {
-  DOUBLE_ENTER_ON_LIST: 'DOUBLE_ENTER_ON_LIST',
   DRAFTJS_BLOCK_KEY: 'DRAFTJS_BLOCK_KEY',
   DRAG_BUTTON_ID: 'button-drag-block',
   INSERT_BLOCK_MARKER_ID: 'insert-block-marker',
   POSITION_ABOVE: 'POSITION_ABOVE',
-  POSITION_AFTER: 'POSITION_AFTER',
-  POSITION_BEFORE: 'POSITION_BEFORE',
   POSITION_BELOW: 'POSITION_BELOW',
-  SOFT_NEWLINE: 'SOFT_NEWLINE',
+
+  ATOMIC_BLOCKS_COPIED: Symbol('ATOMIC_BLOCKS_COPIED'),
```

Comment:
@arunkarnati i agree its not _really_ necessary, but it is _definitely_ cool and we already use `Symbols` for stuff https://github.com/triniti/app-js/blob/ee3656e7f74a398e1ae9f89de3cbf3dffe86d817/src/events/FormEvent.js#L9

![Screen Shot 2020-03-05 at 5 38 50 PM](https://user-images.githubusercontent.com/6315388/76041897-37df9c80-5f08-11ea-9b4a-5be6db0b4cb2.png)

but if you really hate it ill revert

Reply:
@efjacobson - not that I hate it, I agree it is cool, but it just adds additional code that's not necessary. 

IMHO, the `Symbol` usage makes sense when you want to avoid accidental overwrites, especially when writing code for a package, in which case you don't know how that code will be used. 

In the example you mentioned, we don't want the third-party code to overwrite the properties (name, data, etc.) while in this case, we know those constants will remain constants and we have full control on them

Reply:
@arunkarnati ok, ok...reverted